outsettingsvalues/templates/outsettingsvalues/index.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Settings</title>
+    {% load custom_filter %}
+</head>
+<body>
+<form action="{% url 'base' %}" method="GET">
+    {#    {{ form }}#}
+    <input type="text" name="setting_name" value="Enter setting name">
+    <button type="submit">Send</button>
+</form>
+
+<ul>
+    {% for item_key, item_val in settings_list.items %}
+        <li>{{ item_key }}:
+            {% if item_val.items %}
+                <ul>
+                    {% checktype item_val as check %}
+                    {% if check == True %}
+                        {% for i_key, i_val in item_val.items %}
+                            <li>{{ i_key }}:
+                                {% if i_val.items %}
+                                    <ul>
+                                        {% for j_key, j_val in i_val.items %}
+                                            <li>{{ j_key }}: {{ j_val }}</li>
+                                        {% endfor %}
+                                    </ul>
+                                    {% else %}{{ i_val }}
+                                {% endif %}
+                            </li>
+                        {% endfor %}
+                    {% else %}
+                        {% for i in item_val %}
+                            <li>{{ i }}</li>
+                        {% endfor %}
+                    {% endif %}
+                </ul>
+            {% else %}
+                {% checktypelist item_val as check %}
+                {% if check == True %}
```

Comment:
(Same to 24 string)
Can use "if" without comparison with True?
```suggestion
                {% if check == True %}
```